app/mailers/teacher_mailer.rb

```diff
@@ -11,44 +11,57 @@ class TeacherMailer < ApplicationMailer
   def welcome_email(teacher)
     @teacher = teacher
     set_body
-    mail to: teacher.email_name,
+    set_recipients
+    mail to: @recipients, # ActionMailer accepts comma-separated lists of emails
          cc: CONTACT_EMAIL,
          subject: email_template.subject
   end
 
-  def deny_email(teacher, reason)
+  def deny_email(teacher, denial_reason)
     @teacher = teacher
-    @reason = reason
+    @denial_reason = denial_reason
     set_body
-    mail to: @teacher.email_name,
+    set_recipients
+    mail to: @recipients,
          cc: CONTACT_EMAIL,
          subject: email_template.subject
   end
 
-  def request_info_email(teacher, reason)
+  def request_info_email(teacher, request_reason)
     @teacher = teacher
-    @reason = reason
+    @request_reason = request_reason
     set_body
-    mail to: @teacher.email_name,
+    set_recipients
+    mail to: @recipients,
          cc: CONTACT_EMAIL,
          subject: email_template.subject
   end
 
   def form_submission(teacher)
     @teacher = teacher
     set_body
+    set_recipients
     if @teacher.not_reviewed?
-      mail to: CONTACT_EMAIL,
+      mail to: @recipients,
            subject: email_template.subject
     end
   end
 
+  # teacher form submission works exactly the same as admin form submission,
+  # but this method name needs to be explicitly included to ensure that the
+  # title inclusion validation passes when updating the EmailTemplate database
+  def teacher_form_submission(teacher)
+    form_submission(teacher)
+  end
+
   private
   def liquid_assigns
     base_rules = {
       bjc_password: Rails.application.secrets[:bjc_password],
       piazza_password: Rails.application.secrets[:piazza_password],
-      reason: @reason
+      # TODO: Review if below two are needed, or can they be refractored?
```

Comment:
clean up: remove the comment or move the comment before the function declaration and add the note